src/lib/php/Dao/UploadDao.php

```diff
@@ -241,6 +241,25 @@ public function getStatus($uploadId, $groupId)
     }
   }
 
+
+    /**
+   * @brief unused function
+   */
+  public function getAssigneeId($uploadId, $groupId)
+  {
+    if ($this->isAccessible($uploadId, $groupId)) {
+      $row = $this->dbManager->getSingleRow("SELECT assignee FROM upload_clearing WHERE upload_fk = $1", array($uploadId));
```

Comment:
Please add the `group_fk` to the `WHERE` clause as well. You can refer to https://github.com/fossology/fossology/pull/1897

Reply:
Added